### PR TITLE
documentation and resource updates

### PR DESCRIPTION
**Pull request by handwerkerd**
In merging #1113, I remembered that https://tedana.readthedocs.io/en/stable/ only updates when we release a new version. Since we're hoping to release a version in the not-to-distant future, I'll use that as motivation to encourage some updates. I opened this PR as a branch on our main repo instead of on my fork, which means @n-reddy @goodalse2019 and others should be able to push to it directly rather than making suggestions through code review.

The changes to the documentation so far include:

- multi-echo.rst
  - Recommending using the earliest echo for head motion correction
  - Updating links to videos (one new video & fixed a few dead links)
  - 2017 OHBM talks don't seem to be online anymore so I removed those links. There should be link to our 2018 multi-echo symposium (through https://www.pathlms.com/ohbm/courses/15243/sections/25422# ) but the link is currently dead

### CODE REVIEW

**Review by tsalo — APPROVED** [review]
LGTM!

EDIT: Sorry, I just realized this is still a draft PR.

**Comment by handwerkerd** [comment]
@tsalo You are welcome to use this PR add some of the newer open datasets to the list in our docs.

**Comment by handwerkerd** [comment]
Note to self: I want to update the posters on our front page https://tedana.readthedocs.io/en/stable/ and read through that page's text to see if there are others things worth updating there.

**Comment by goodalse2019** [comment]
I've gone through the title page - all looks good to me! The external regressors piece is a subsection of the "defining a custom decision tree" so no changes needed there unless you were wanting to highlight that change specifically! Updating posters, a good idea.

File: docs/index.rst [hunk 1]
```diff
@@ -112,8 +112,8 @@ When using tedana, please include the following citations:
       }
       </script>
       <p>
-      <span id="tedana_citation">tedana</span>
-      This link is for the most recent version of the code and that page has links to DOIs
+      <span id="tedana_citation"></span>
+      1. This link is for the most recent version of the code and that page has links to DOIs
       for older versions. To support reproducibility, please cite the version you used:
       <a id="tedana_doi_url" href="https://doi.org/10.5281/zenodo.1250561">https://doi.org/10.5281/zenodo.1250561</a>
```
Comment (handwerkerd) [inline docs/index.rst:118]: This isn't an issue for this PR, but it looks like there aren't zenodo DOIs for tedana versions past 23.0.1. @tsalo Do you know who/how to add version to zenodo and where in our new release workflow that should be happening?

Comment (tsalo) [reply docs/index.rst:118]: It looks like the Zenodo setting was somehow disabled. I just re-enabled it, so it should work the next time we make a release.

**Review by n-reddy — APPROVED** [review]
LGTM!

**Review by tsalo — APPROVED** [review]

**Pull request merged**